usermembership/tool/src/bundle/org/sakaiproject/umem/tool/bundle/Messages.properties

```diff
@@ -63,3 +63,6 @@ deselect_all=Deselect All
 export_selected_to_csv=Export Selected to CSV
 export_selected_to_excel=Export Selected to Excel
 actions_header=Actions
+
+date_format=MMM d, yyyy
```

Comment:
Isn't there a better way to handle this, rather than having the date format in the i18n strings?

Reply:
Forum Tool:
<f:convertDateTime pattern="#{msgs.date_format}" timeZone="#{ForumTool.userTimeZone}" locale="#{ForumTool.userLocale}"/>
?

Reply:
Yeah, but then we end up with everyone having to put the date format in their localisation files and less maintained locales end up with bugs where it didn't get done. Can't we just use the built in locale formats?

`<c:convertDateTime dateFormat="short">